### ISSUE TITLE
Upgrading zone.js to latest ^0.6.18 doesn't let app build

### ISSUE THREAD

**Issue by roblav96** [NativeScript/NativeScript#2700]
## tldr;
Revert back to `0.6.17` zone.js

---

I get this error when trying to build my app:
```
Sep 10 12:27:29 Robs-MBP client[10264]: 3   0x10b85392d start
Sep 10 12:27:29 Robs-MBP client[10264]: **file:///app/tns_modules/zone.js/dist/zone-node.js:670:21: JS ERROR Error: Could not find module 'timers'**. Computed path '/Users/roblav96/Library/Developer/CoreSimulator/Devices/F0820311-799D-4113-AE34-3C3AAD669E9F/data/Containers/Bundle/Application/B5D1EF38-393A-4AA2-B204-C03023FC42D6/client.app/app/tns_modules/timers'.
Sep 10 12:27:29 Robs-MBP com.apple.CoreSimulator.SimDevice.F0820311-799D-4113-AE34-3C3AAD669E9F.launchd_sim[2436] (UIKitApplication:com.roblav96.sandbox[0x4bbd][10264]): Service exited due to signal: Segmentation fault: 11
```

Checking `npm info zone.js`:
```
     '0.6.16': '2016-08-22T17:29:49.463Z',
     '0.6.17': '2016-08-22T22:41:52.196Z',
     '0.6.18': '2016-09-10T16:04:53.468Z',
     '0.6.19': '2016-09-10T16:06:17.234Z' },
```

I revert back to `0.6.17` and everything works fine again.

**Comment by warren-bank** [comment]
ditto.. I'm not sure whether the zone.js team would necessarily care about code changes that would break compatibility with NativeScript, but I added an [issue](https://github.com/angular/zone.js/issues/437) in their repo that describes the problem and its origin.

In the meanwhile, we're stuck with either:
* reverting our version of zone.js (as suggested here) to a known/working copy
* write a (temporary?) workaround that exposes a module "timers", which simply wraps the existing module ["timer"](https://github.com/NativeScript/NativeScript/tree/master/tns-core-modules/timer)

**Comment by ross-bradbury** [comment]
This looks related to [nativescript-angular issue #144](https://github.com/NativeScript/nativescript-angular/issues/444)

**Comment by warren-bank** [comment]
### tldr; (just to summarize how to revert version.. step-by-step)

```bash
tns create ExampleProject --ng
cd ExampleProject/node_modules/nativescript-angular
npm uninstall zone.js --save
npm install zone.js@0.6.17 --save

cd ../..
rm -rf platforms
tns run android
```

**Comment by warren-bank** [comment]
update: I was curious.. and learned something new..

the module: `@angular/core` is a ["scoped package"](https://docs.npmjs.com/misc/scope)

it's ["package.json"](https://github.com/angular/angular/blob/master/modules/%40angular/core/package.json) includes "zone.js" in its "peerDependencies".

when `npm` installs `@angular/core`, the peer is added adjacent to the parent "scope" (ie: as a peer to `@angular`), which is the 2nd instance of "zone.js" reverted in the above example.

**Comment by tsonevn** [comment]
related to issue [#444](https://github.com/NativeScript/nativescript-angular/issues/444) .
Please comment in issue  [#444](https://github.com/NativeScript/nativescript-angular/issues/444) .
